### DISCUSSION TITLE
Help us donate for Astroid

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#705] [Announcements]
Dear all,

This is the first time I have made a call for donations for Astroid. Until now, we have tried to maintain this project as a GPL project for the community. And through this call for donations, I hope to achieve the following goals:

1. Call for a source of funding to continue developing Astroid Framework
2. Estimate the size and strength of the Astroid community. From there, take appropriate steps in the next versions.

To donate to Astroid visit this link https://ko-fi.com/astroidframework/goal?g=0

With my best regards,
Sonny

**Comment by sonvnn** [comment]
@Banuk1616 Thank you for your coffees :)

**Comment by Chacapamac** [comment]
Ok, I’m in, I send you 30$ but I will send more....

**Comment by Chacapamac** [comment]
To everyone here, You know how the developers of the BEST Joomla Framework work hard to give YOU this fantastic Tool. I’m just a user and also trying to help Astroid (when I can...). I recognize how much love and work is made every day for all of us using it. Buying a coffee, especially to @sonvnn and his dedication to this great software is the minimum we can do in return! 🚀 — **Let’s reach the Goal!**

**Reply by Banuk1616** [reply]
I see it the same way, that's why I gave what I could at the moment and I'll give more when I can again. I think everyone who uses Astroid should do this. Not just take but also give.

**Reply by sonvnn** [reply]
Thank you all. I will try my best to maintain and upgrade Astroid to the best Joomla Framework.

Best regards,
Sonny

**Reply by Chacapamac** [reply]
> Thank you all. I will try my best to maintain and upgrade Astroid to the best Joomla Framework.

You already did @sonvnn 
**Thank You!** 👍

**Comment by Banuk1616** [comment]
It's been 6 days since Sonny launched the appeal for donations for the Astroid Framework and so far only four people have made a donation. Considering the number of users asking for help etc. in the forum, this is really sad. Many users use the Astroid Framework, as I do, only for private websites or for the website of the association in which they are active, without earning money with it. Other users earn money with the templates they have created with the Astroid Framework and still other users may earn money with the websites they have created with the Astroid Framework. Sonny invests his free time in the Astroid Framework and makes it available to us free of charge. But then it should also be possible for everyone who uses the Astroid Framework to donate one or more coffees, right?
Guys, come on, don't be so stingy. Anyone who runs a website based on the Astroid Framework, or develops and sells templates with the Astroid Framework, or earns money with websites created with the Astroid Framework, can also support the Astroid Framework and honor Sonny's work with a donation.

One should not always just take but also give something back. Sonny gives us the Astroid Framework, which for me is the best framework in combination with Joomla, what do you give back?

**Comment by Chacapamac** [comment]
Completely agree, I will continue to donate from time to time. Common Guys and Girls, Devs here give a lot of their time on this. If you liked Astroid like I do, give a little, it will show that you care. 💕  🚀

**Comment by Banuk1616** [comment]


**Reply by sonvnn** [reply]
Thanks for supporting Astroid. I do this for love and joy. Don't worry I'll try to keep it going as long as I can.

**Comment by paurray** [comment]
I have seen so much wonderful software bite the dust and do not want to see this happen to Astroid. I made a small symbolic contribution as a token of my gratitude and will do so again in the future.

**I can only urge anyone else who digs Astroid to do the same.**

Thank you Sonny.
